pkg/controller/metrics/metricscontroller.go

```diff
@@ -0,0 +1,274 @@
+// Copyright (c) 2017 Huawei Technologies Co., Ltd. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
+This module implements a entry into the OpenSDS volume controller service.
+
+*/
+
+package metrics
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	//"errors"
+	"fmt"
+
+	"github.com/opensds/opensds/pkg/dock/client"
+	"github.com/opensds/opensds/pkg/model"
+	pb "github.com/opensds/opensds/pkg/model/proto"
+)
+
+// Controller is an interface for exposing some operations of different volume
+// controllers.
+type Controller interface {
+
+    GetLatestMetrics(opt *pb.GetMetricsOpts) (*[]model.MetricSpec, error)
```

Comment:
Please use `gofmt` to update the code style.

Reply:
done